backend/lcfs/web/api/fuel_export/repo.py

```diff
@@ -392,5 +392,5 @@ async def get_effective_fuel_exports(
         result = await self.db.execute(query)
         fuel_exports = result.unique().scalars().all()
 
-        logger.debug(f"Retrieved {len(fuel_exports)} effective FuelExport records.")
+        #logger.debug(f"Retrieved {len(fuel_exports)} effective FuelExport records.")
```

Comment:
there are a number of commented out logger lines, we should probably just remove them.

Reply:
Deleted